mods/cnc/languages/lua/en.ftl

```diff
@@ -150,3 +150,13 @@ nod-soldier = Nod Soldier
 civilians-runs = Hey, those civilians... where are they going?
 destroy-ion-cannon-advise = The GDI are preparing their ion cannon. Don't let them get used to it.
 village-destruction-warning = Be careful, commander. The GDI won't stand still while we burn the entire village.
+
+## twist-of-fate
+clear-path = Repel the ambush and clear the way
+    for our MCV.
+recover-old-base = Capture the Construction Yard in our recon
+    post to regain control.
+air-strikes-intel-report = Nod airstrikes are being directed by a Communications Center, located northwest. We may gain useful information from its capture.
+capture-nod-communications-center = Capture the Nod Communications Center to
+    the northwest.
+communications-center-captured-sams-located = Our engineers located Nod SAM sites. They also disarmed an unusual trap on the Construction Yard to the south.
```

Comment:
I guess we should remove the SAM location revealing too now.
```suggestion
communications-center-captured = Communications Center captured. Our engineers managed to disarm an unusual trap on the Construction Yard to the south.
```
and remove the SAM revealing logic?

Reply:
I think it's fine to keep it

Reply:
Hm, I guess we could keep it so that the player can maneuver their Orcas a bit more freely, but I still think it is weird.